src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -17992,5 +18012,277 @@ public void InterpolatedStringInForeach_HasErrors()
                   NextVariables(0)
                 """);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/66235")]
+        public void ConvertStringToInterpolatedStringHandlerError_DefaultInterpolatedStringHandler_ByVal()
```

Comment:
The original intent was to verify that behaviour is equvivalent for custom-made handler and a default one since `DefaultInterpolationStringHandler` is somewhat special type for the compiler. But if you don't see any value in such checks, I am happy to remove unneded tests